View_Assist_custom_sentences/Play_Music_with_Music_Assistant/blueprint-playmusicwithmusicassistant.yaml

```diff
@@ -24,152 +24,188 @@ blueprint:
       description: The View Assist dashboard view to use for displaying music information (example /dashboard-viewassist/music)
       default: "/dashboard-viewassist/music"
 alias: View Assist - Play Music with Music Assistant
-description: ""
+description: "Play music using music assistant"
 trigger:
   - platform: conversation
     command:
       - play the artist {artist}
-      - play some {artist}
-    id: playartist
+      - play some {artist} [music]
+      - play [some] [music | songs] by {artist}
+    id: artist
```

Comment:
I'm not sure why you changed all of the ids.   Change them back to original for playartist, playsong, cuesong